data/json/recipes/recipe_deconstruction.json

```diff
@@ -1320,12 +1320,12 @@
         [ [ "tazer", 1 ] ],
         [ [ "m4a1", 1 ] ],
         [ [ "motor_large", 1 ] ],
-        [ [ "power_supply", 30 ] ],
+        [ [ "power_supply", 20 ] ],
         [ [ "plut_cell", 4 ] ],
-        [ [ "processor", 3 ] ],
-        [ [ "RAM", 4 ] ],
-        [ [ "steel_chunk", 20 ] ],
-        [ [ "mil_plate", 2 ] ]
```

Comment:
I tried to make the usefulness similar rather than going by weight. If weight was to be retained, the tankbot would be disassembled to 300 steel chunks rather than 30.